subsys/net/lib/aws_jobs/src/aws_jobs.c

```diff
@@ -523,7 +534,7 @@ static int construct_job_id_update_topic(const u8_t *client_id,
 			   client_id,
 			   job_id,
 			   suffix);
-	job_id_update_topic_len = ret + 1;
```

Comment:
This is a patch for `aws_jobs` as adding 1 would cause the incorrect length as we don't care about the delimiter when using the utf-8 struct in the mqtt library.